Assets/TowerDefense/Scripts/ElementTowers/ObjectsFloatingRandomlyAnimation.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ObjectsFloatingRandomlyAnimation : MonoBehaviour
+{
+    public List<ObjectRotator> Objects;
+    public ObjectRotator ObjectRotator;
```

Comment:
z kodu wynika, że to pole nie powinno być przypisane w inspektorze - użyj atrybutu [HideInInspector]

Reply:
Raczej tak chcialem
do Objects z inspektora wrzucam prefaby do losowania do animacji a ObjectRotator to prefab ktory tez przypisuje w inspektorze ktory rotuje obiektami z Objects